{"name": "test_empty[hcodes10]", "status": "failed", "statusDetails": {"message": "AssertionError: assert [('00149', 'Т... 4, ...), ...] == []\n  Left contains 130 more items, first extra item: ('00149', 'Темп роста пр. труда работников, занятых на перевозках', '%', 480, 17, 4, ...)\n  Full diff:\n    [\n  +  ,\n  -  ('00149',\n  -   'Темп роста пр. труда работников, занятых на перевозках',\n  -   '%',...\n  \n  ...Full output truncated (2209 lines hidden), use '-vv' to show", "trace": "connection = <cursor object at 0x03B7B128; closed: 0>, hcodes = ('00155', '00026', '00149')\nquery_3_03 = 'create or replace function public.lilo_auto_qa() \\n\\treturns table(\\thcode_id bpchar, hcode_name varchar(300), hcode_....kato_id \\n\\t\\tand a.dir_id = b.dir_id\\ncross join (select * from dm.d_hcode_t where id = %s) hcd\\n    ;\\nend;\\n$f$\\n;'\nquery_3_04 = \"select * from public.lilo_auto_qa()\\nexcept\\nselect  hcode_id, hcode_name, hcode_unit_name, org_id, dor_kod, date_typ...uch_id, nod_id, dir_id, kato_id, vids_id from dm_rep.dm_all_indicators_v\\nwhere hcode_id = %s\\nand calc_rule = 'ratio'\"\n\n    @allure.feature('Расчетные показатели')\n    @allure.story('Полная сверка')\n    @allure.severity('critical')\n    @pytest.mark.parametrize('hcodes',hcodes)\n    def test_empty(connection, hcodes , query_3_03, query_3_04):\n        connection.execute(query_3_03, (hcodes[0], hcodes[1], hcodes[2], hcodes[2]))\n        connection.execute(query_3_04, (hcodes[2],))\n        lines = connection.fetchall()\n        connection.execute('drop function public.lilo_auto_qa();')\n    \n>       assert lines == []\nE       AssertionError: assert [('00149', 'Т... 4, ...), ...] == []\nE         Left contains 130 more items, first extra item: ('00149', 'Темп роста пр. труда работников, занятых на перевозках', '%', 480, 17, 4, ...)\nE         Full diff:\nE           [\nE         +  ,\nE         -  ('00149',\nE         -   'Темп роста пр. труда работников, занятых на перевозках',\nE         -   '%',...\nE         \nE         ...Full output truncated (2209 lines hidden), use '-vv' to show\n\ntest_income.py:42: AssertionError"}, "attachments": [{"name": "stdout", "source": "1cf7c878-751d-4db8-b9d9-9458f79a01ce-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "hcodes", "value": "('00155', '00026', '00149')"}], "start": 1604502564370, "stop": 1604502590250, "uuid": "34b3a976-9fb0-4003-9f55-da4bc42aee21", "historyId": "725b96966f25c29fb7413cadeb5829fd", "testCaseId": "07ce32e44c681d301be8a13e0c006f7d", "fullName": "test_income#test_empty", "labels": [{"name": "severity", "value": "critical"}, {"name": "feature", "value": "Расчетные показатели"}, {"name": "story", "value": "Полная сверка"}, {"name": "suite", "value": "test_income"}, {"name": "host", "value": "62032-nb"}, {"name": "thread", "value": "1768-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_income"}]}